internal/tiltfile/tiltfile_test.go

```diff
@@ -4798,6 +4798,48 @@ k8s_resource('foo', objects=['baz:namespace:qux'])
 	f.loadErrString("No object identified by the fragment \"baz:namespace:qux\" could be found. Possible objects are: \"foo:Deployment:default\", \"baz:Namespace:default\"")
 }
 
+func TestK8sResouceObjectsNonWorkloadOnly(t *testing.T) {
```

Comment:
also what's the meaning of the "Only" in this test name? I think I get that "workload only" means "no additional objects passed" but this usage doesn't make sense to me at first glance.

Reply:
It's *Non*WorkloadOnly, so only non-workload objects are passed